src/main/factsManager.js

```diff
@@ -36,7 +36,7 @@ fluid.defaults("gpii.app.factsManager", {
         keyedInTimestamp: null,
         // Number of interactions with the GPII app. Incremented whenever the PSP or QSS is
         // opened, as well as when an actual user (i.e. different from `noUser`) keys in.
-        interactionsCount: 0
+        interactionsCount: null
```

Comment:
Why was this default changed?

Reply:
My idea was not to mislead the reader because now the `interactionsCount` is loaded from a file when the application starts up. In most cases the initial value will not be 0 but what it was when the user last interacted with the GPII app.